src/Config/Loader.php

```diff
@@ -2,33 +2,48 @@
 
 namespace PharIo\ComposerDistributor\Config;
 
+use DOMDocument;
+use RuntimeException;
+
 class Loader
 {
-    /**
-     * Config json example:
-     *
-     * "keyDirectory" and "signature" are optional
-     *
-     * {
-     *   "packageName": "phar-io/phive",
-     *   "keyDirectory: "keys",
-     *   "phars": [
-     *     {
-     *       "name": "phive",
-     *       "file:" "https://github.com/phar-io/phive/releases/%version%/phive.phar",
-     *       "signature:" "https://github.com/phar-io/phive/releases/%version%/phive.phar.asc"
-     *     }
-     *   ]
-     * }
-     *
-     */
     public static function loadFile(string $configFile): Config
     {
         if (!is_file($configFile)) {
-            throw new \RuntimeException('Config file is missing');
+            throw new RuntimeException('Config file is missing');
```

Comment:
Do we actually want to consider throwing our own Exceptions instead?

Reply:
Own exceptions ✅